src/css.js

```diff
@@ -113,6 +113,18 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
```

Comment:
> > It can even store the `window.top !== window.self` bit

> Hmm, I wonder if we need to handle the case where a jQuery in one frame is manipulating elements in another frame. We do get occasional bug reports about those use cases. If we wanted to handle that we would need to use `elem.ownerDocument` and `elem.document.defaultView` each time?

To maximize internal consistency, I recommend matching [Sizzle](https://github.com/jquery/sizzle/blob/master/src/sizzle.js#L509): `(parent = document.defaultView) && parent.top !== parent`, where `document` is `elem.ownerDocument`. Unless, of course, this bug also seeps into child frames, in which case it should be `window.top !== window`. In either case, though, `self` is redundant.